Task1.java

```diff
@@ -0,0 +1,83 @@
+public class Task1 {
+    public interface Figure {
+        public int area();
+    }
+
+    public class Circle implements Figure {
+        public int radius;
+
+        public Circle(int radius) {
+            this.radius = radius;
+        }
+
+        @Override
+        public int area() {
+            return (int) (radius * radius * Math.PI);
+        }
+    }
+
+    public class Triangle implements Figure {
+        public int side;
+        public int height;
+
+        public Triangle(int side, int height) {
+            this.side = side;
+            this.height = height;
+        }
+
+        @Override
+        public int area() {
+            return (side * height) / 2;
+        }
+    }
+
+    public class Square implements Figure {
+        public int side;
+
+        public Square(int side) {
+            this.side = side;
+        }
+
+        @Override
+        public int area() {
+            return side * side;
+        }
+    }
+
+   public class arrFigure {
+       Figure[] figures;
+        private static int counter = 0;
+
+        public arrFigure(int quantity){
+            figures = new Figure[quantity];
+        }
+
+        public void addFigure(Figure figure){
+            if(figure instanceof Figure && figures  != null && figures.length > 0){
```

Comment:
Разве тут может быть какой-либо иной тип кроме как Figure?
```suggestion
            if(figures  != null && figures.length > 0){
```